docker/Dockerfile

```diff
@@ -12,8 +12,15 @@ RUN apk add --no-cache \
 	g++ \
 	bash \
 	git \
-	openssl-dev
+	openssl-dev \
+    icu-libs \
+    libintl \
+    icu \
+    icu-dev \
+    libxml2-dev
```

Comment:
Please use alpha sort and use the same indention as line 15

Reply:
Had some stray tabs.